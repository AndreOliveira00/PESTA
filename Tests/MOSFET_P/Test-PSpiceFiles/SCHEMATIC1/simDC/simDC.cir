** Profile: "SCHEMATIC1-simDC"  [ C:\Users\Andre\Desktop\Test\Test-PSpiceFiles\SCHEMATIC1\simDC.sim ] 

** Creating circuit file "simDC.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../test-pspicefiles/test.lib" 
* From [PSPICE NETLIST] section of C:\Users\Andre\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 1000ns 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
