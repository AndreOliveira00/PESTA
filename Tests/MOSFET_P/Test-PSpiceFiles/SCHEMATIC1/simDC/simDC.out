
**** 06/02/21 09:26:56 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-simDC"  [ C:\Users\Andre\Desktop\Test\Test-PSpiceFiles\SCHEMATIC1\simDC.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "simDC.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../test-pspicefiles/test.lib" 
* From [PSPICE NETLIST] section of C:\Users\Andre\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 1000ns 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST
R_R1         0 N21268  10k TC=0,0 
C_C1         0 N14709  4.7u  TC=0,0 
R_RL         0 N14709  100R TC=0,0 
V_V1         N14763 0 24Vdc
X_Q1         N14763 N21268 N14709 Si1433DH
D_D1         N21268 N14709 ZY15GP 

**** RESUMING simDC.cir ****
.END

**** 06/02/21 09:26:56 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-simDC"  [ C:\Users\Andre\Desktop\Test\Test-PSpiceFiles\SCHEMATIC1\simDC.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               ZY15GP          X_Q1.DBD        
          IS   10.000000E-15    1.000000E-12 
         ISR  395.250000E-09                 
          BV   13.801          30.2          
         IBV     .051734                     
          RS    1.000000E-03     .1          
          TT    5.000000E-09   60.000000E-09 
         CJO    1.000000E-12   60.000000E-12 
          VJ     .75             .38         
           M     .3333           .38         


**** 06/02/21 09:26:56 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-simDC"  [ C:\Users\Andre\Desktop\Test\Test-PSpiceFiles\SCHEMATIC1\simDC.sim ] 


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               X_Q1.PMOS       X_Q1.NMOS       
               PMOS            NMOS            
       LEVEL    3               3            
         TPG   -1              -1            
           L  100.000000E-06  100.000000E-06 
           W  100.000000E-06  100.000000E-06 
         VTO   -3.820649        3.213299     
          KP   50.000000E-09   41.437730E-06 
       GAMMA    3.121473        2.502746     
         PHI     .831948         .809092     
      LAMBDA    0               0            
          RS     .04                         
          IS    0              10.000000E-15 
          JS    0               0            
          PB     .8              .8          
        PBSW     .8              .8          
          CJ    1.205117E-03  966.243300E-06 
        CJSW    0               0            
        CGSO    0               0            
        CGDO    0               0            
        CGBO    0               0            
        NSUB  140.000000E+15   90.000000E+15 
         NFS  800.000000E+09    1.000000E+12 
         TOX   50.000000E-09   50.000000E-09 
          XJ  500.000000E-09    0            
          UO  400                            
       UCRIT   10.000000E+03   10.000000E+03 
       DELTA     .1                          
         ETA  100.000000E-06                 
       KAPPA     .11                         
      DIOMOD    1               1            
         VFB    0               0            
        LETA    0               0            
        WETA    0               0            
          U0    0               0            
        TEMP    0               0            
         VDD    5               5            
       XPART    0               0            


**** 06/02/21 09:26:56 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-simDC"  [ C:\Users\Andre\Desktop\Test\Test-PSpiceFiles\SCHEMATIC1\simDC.sim ] 


 ****     Resistor MODEL PARAMETERS


******************************************************************************




               X_Q1.RTEMP      
           R    1            
         TC1    7.000000E-03 
         TC2    5.500000E-06 


**** 06/02/21 09:26:56 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-simDC"  [ C:\Users\Andre\Desktop\Test\Test-PSpiceFiles\SCHEMATIC1\simDC.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N14709)   23.3750 (N14763)   24.0000 (N21268)    9.6765 (X_Q1.3)   23.9900 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1        -2.347E-01

    TOTAL POWER DISSIPATION   5.63E+00  WATTS



          JOB CONCLUDED

**** 06/02/21 09:26:56 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-simDC"  [ C:\Users\Andre\Desktop\Test\Test-PSpiceFiles\SCHEMATIC1\simDC.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  License check-out time            =       14.96
  Total job time (using Solver 1)   =         .06
